### ISSUE TITLE
Term issue: Restriction enzymes

### ISSUE THREAD

**Issue by nataled** [PROconsortium/PRoteinOntology#157]
@mikebada I recall you mentioning that you needed to have these for your text tagging. I've added many of them to PRO. I added all subunits (methylase, endonuclease, etc) but I suspect you only need the endonucleases, correct? If so, I'll produce a list giving the enzyme and the corresponding PRO identifier.

**Comment by mikebada** [comment]
Thanks much.  I've compiled a list of restriction enzymes mentioned in CRAFT that we weren't previously able to find in PRO.  I've searched for the first ten or so in PRO by their abbreviations now and haven't been able to find them yet either.  I'll e-mail you this list so that you can have it in a more convenient document form.

**Comment by nataled** [comment]
Hi Mike, 
I should have warned you that you'll not be able to search for these on the web since they haven't yet been made public. Nonetheless, there were indeed many missing (since I only processed those with corresponding Swiss-Prot entries and not TrEMBL). I've now completed all requested items (including those with no known sequence). See mapping below:

AatII PR:P70725
AgeI PR:Q9KHV6
ApaI PR:E3VXA1
AscI PR:E3VXA3
AseI PR:E3VXA5
BclI PR:E5LGB8
BglI PR:O68557
BglII PR:Q45488
BssHII PR:O30876
ClaI PR:F1KC47
EagI PR:B0LT74
EcoRI PR:P00642
EcoRII PR:P14633
EcoRV PR:P04390
FseI PR:F1KC51
FspI PR:F1KC58
HaeIII PR:O68584
HhaII PR:P00643
HincII PR:P17743
HinP1I PR:Q5I6E6
HpaI PR:P29537
HpaII PR:P36433
KpnI PR:P25237
MluI PR:G3EST6
MspI PR:P11405
NaeI PR:P50187
NcoI PR:O85489
NheI PR:Q6S994
NotI PR:Q2I6W2
NsiI PR:C6M5V9
PacI PR:D5MNX7
PmeI PR:A0A2R3QXB2
PstI PR:P00640
PvuII PR:P23657
SacI PR:O31074
SacII PR:J9UZ90
SalI PR:Q53608
SapI PR:O52712
ScaI PR:O52691
SfiI PR:O52512
SmaI PR:P14229
SpeI PR:F1KM35
SphI PR:P95821
StuI PR:F1KM37
SwaI PR:A0A1S4NYF7
XbaI PR:O68567
XhoI PR:Q9KVZ7
XmaI PR:O68566
BbvCI PR:000050109
Asp718I PR:000050201
Bsp120I PR:000050202
Bsu36I PR:000050203
KflI PR:000050204
KspI PR:000050205
MaeII PR:000050206
MfeI PR:000050207
NdeI PR:000050208
PmlI PR:000050209
PpuMI PR:000050210
PspOMI PR:000050211
RsrII PR:000050212
SanDI PR:000050213
SstI PR:000050214
SrfI PR:000050215
